internal/store/files/recog_place_map.go

```diff
@@ -99,5 +116,28 @@ func RecogPlaceMap() (map[string]*pb.RecogPlaces, error) {
 		}
 	}
 
-	return recogPlaceMap, nil
+	// Add more names in the pattern of "selfName containingPlaceName", e.g., "Brussels Belgium".
```

Comment:
I slightly wonder about this.  IIRC, the existing algorithm already should infer that <Brussels> is a place, and then <Belgium> is a place, and further <Brussels> is in <Belgium> so we can decide that we should consume both tokens to match Brussels?

Reply:
Right, I thought about the capability of existing algorithms. However, if using the capability of existing algorithms, then:
1. We need to add the expanded name to RecogizePlacesResponse, which is not needed for RecognizePlaces users.
2. The existing algorithm is already complicated, plugging in more logic there will make it a bit more complex. In comparison, memorizing dcidToNames in store is not computationally-heavy, but could leave RecognizePlaces code intact.

Reply:
Ah, I missed that DcidToNames is only used in FindEntities flow.  This is good!